.eslintrc.yml

```diff
@@ -17,6 +17,8 @@ plugins: [react, prettier]
 rules:
   jsx-a11y/click-events-have-key-events: 'warn'
   jsx-a11y/no-noninteractive-element-interactions: 'warn'
+  import/extensions: ["error", "never", { "packages": "always" }]
```

Comment:
I have added `import/extensions: [error, {"js": never, "jsx": never}]` as it is in climate watch, we can combine those two: 
`import/extensions: [error, {"js": never, "jsx": never, "packages": always}]`

Reply:
👍